DigitalLearningSolutions.Web/Controllers/TrackingSystem/CentreConfiguration/RegistrationPromptsController.cs

```diff
@@ -261,6 +304,19 @@ private IActionResult AddRegistrationPromptConfigureAnswersPostNext(Registration
             return RedirectToAction("AddRegistrationPromptSummary");
         }
 
+        private IActionResult TryRemoveRegistrationPrompt(int promptNumber)
+        {
+            try
+            {
+                customPromptsService.RemoveCustomPromptFromCentre(User.GetCentreId(), promptNumber);
+                return RedirectToAction("Index");
+            }
+            catch
+            {
+                return RedirectToAction("Error", "LearningSolutions");

```

Comment:
I think the try catch -> redirect to error is redundant here as that should be the automatic behaviour.

Reply:
The default behaviour logs an error too, which we should be doing! I think either logging a sensible error here that we can use for debugging in addition to the redirect, OR letting it bubble are both something we can do :)